### PR TITLE
Adding method for merging Swagger YAML files

### PR DESCRIPTION
**Pull request by danqrodney**

### CODE REVIEW

File: swagger.rb [hunk 1]
```diff
@@ -12,8 +13,23 @@ def download(url, path)
         IO.copy_stream(open(url), f)
       end
     end
-    module_function :download
-
+    module_function :download    
+    
+    def merge_yaml(main_yaml_file, files_to_include, output_yaml_file)
```
Comment (calbach) [inline swagger.rb:18]: Have you verified that this produces deterministic output?

**Review by calbach — APPROVED** [review]

File: swagger.rb (hunk 1 above)
Comment (danqrodney) [reply swagger.rb:18]: I haven't, but I'm not sure it matters? (As long as the contents of the YAML are equivalent... we aren't checking in the generated file anywhere...)

Comment (calbach) [reply swagger.rb:18]: I wouldn't be surprised if non-deterministic swagger == non-deterministic codegen (at least in some languages). I guess so long as the Python codegen output is deterministic, this isn't a major issue.

Comment (danqrodney) [reply swagger.rb:18]: So at the moment, the only thing using the merged YAML will be our server and UI codegen, which aren't checked in anywhere... so if the order of things changes it doesn't really matter.

If we start using this merge method for our client APIs, having it result in stuff in the same order seems more important.

**Pull request merged**